templates/docs/patterns/accordion.md

```diff
@@ -36,6 +36,12 @@ View example of the accordion pattern
 View example of the accordion pattern
 </a></div>
 
+**With tick elements:**
```

Comment:
As this is a separate variant requiring different class name would be good to have it with it's own heading.

Would be good to add a sentence describing when to use it.
```suggestion
## With tick elements

When checkboxes or radio buttons are used inside accordion panels add `has-tick-elements` class name to the panel `p-accordion__panel` element, to properly align tick elements with accordion controls.
```

This will also allow us to link to this directly with anchor link from "What's new" page.

Reply:
Done